test/e2e/specs/editor/various/autocomplete-and-mentions.spec.js

```diff
@@ -0,0 +1,429 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+const userList = [
+	{
+		username: 'testuser',
+		firstName: 'Jane',
+		lastName: 'Doe',
+		password: 'iLoVeE2EtEsTs',
+	},
+	{
+		username: 'yourfather',
+		firstName: 'Darth',
+		lastName: 'Vader',
+		password: 'padme123',
+	},
+	{
+		username: 'mockingjay',
+		firstName: 'Katniss',
+		lastName: 'Everdeen',
+		password: 'district12forlyfe',
+	},
+	{
+		username: 'ringbearer',
+		firstName: 'Frodo',
+		lastName: 'Baggins',
+		password: 'ep1cburgl@r',
+	},
+	{
+		username: 'thebetterhobbit',
+		firstName: 'Bilbo',
+		lastName: 'Baggins',
+		password: 'lostwithouts@m',
+	},
+	{
+		username: 'makeitso',
+		firstName: 'Jean-Luc',
+		lastName: 'Picard',
+		password: 'engagE!1',
+	},
+	{
+		username: 'buddytheelf',
+		firstName: 'Buddy',
+		lastName: 'Elf',
+		password: 'sm1lingsmyfavorite',
+	},
+];
+test.describe( 'Autocomplete', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		for ( const user of userList ) {
+			await requestUtils.createUser( {
+				email: `${ user.username }@example.com`,
+				...user,
+			} );
+		}
```

Comment:
Maybe we can run them in parallel to speed it up a little?

```suggestion
		await Promise.all(
			userList.map( ( user ) => (
				requestUtils.createUser( {
					email: `${ user.username }@example.com`,
					...user,
				} )
			) )
		);
		for ( const user of userList ) {
			await requestUtils.createUser( {
				email: `${ user.username }@example.com`,
				...user,
			} );
		}
```

Reply:
Great idea. Implemented in [3886ba4](https://github.com/WordPress/gutenberg/pull/42905/commits/3886ba462888da6c4f6fafdb479b498a015dfaa3) (the above suggestion still has the original `for` loop, so I couldn't use that one :)

Reply:
Haha oops, I forgot to delete that 😅 .